### PR TITLE
Improve gsc_mann_whitney_de

### PR DESCRIPTION
**Pull request by bellenger-l**
Improve output according to comments in #403
- sorting by p.adjust
- replace T/F vector by up/down/ns in *Significant* column of output

### CODE REVIEW

File: tools/gsc_mannwhitney_de/MannWhitney_DE.R [hunk 1]
```diff
@@ -138,9 +138,14 @@ gene_stats <- descriptive_stats(data.counts)
 results <- merge(gene_stats, MW_test, by = "row.names")
 colnames(results)[1] <- "genes"
 
+## Annotate Significant column
+results$Significant[results$Significant == T & !is.na(results$Significant)] <- ifelse(subset(results, Significant == T)$log2FC > 0, "UP", "DOWN")
+results$Significant[results$Significant == F & !is.na(results$Significant)] <- "NS"
```
Comment (drosofff) [inline tools/gsc_mannwhitney_de/MannWhitney_DE.R]: "NS" or "NA" ??

Comment (bellenger-l) [reply tools/gsc_mannwhitney_de/MannWhitney_DE.R]: NS as "non significant" when p.adjust > FDR

**Comment by bellenger-l** [comment]
Bump version to 0.9.4

Without further return, I will merge this PR after Travis tests

**Comment by drosofff** [comment]
OK @bellenger-l . But never merge without feed back on tools-artbio, please. This is my privilege ;-)

**Comment by bellenger-l** [comment]
Noted ! :)

**Comment by drosofff** [comment]
@bellenger-l you can merge through

**Pull request merged**